ddtrace/_trace/trace_handlers.py

```diff
@@ -126,6 +129,42 @@ def _start_span(ctx: core.ExecutionContext, call_trace: bool = True, **kwargs) -
     return span
 
 
+def _set_web_frameworks_tags(ctx, span, int_config):
+    span.set_tag_str(COMPONENT, int_config.integration_name)
+    span.set_tag_str(SPAN_KIND, SpanKind.SERVER)
```

Comment:
Can we make this function a bit more general? If we set/access the span span_kind from `ctx` then we could use this method in more integrations.

Reply:
We can change this in future iterations if necessary.